packages/frontend/src/common/context/ColorThemeContext.tsx

```diff
@@ -22,8 +30,53 @@ declare module "@mui/material/styles" {
     highlightColorPrimary: string;
     highlightColorSecondary: string;
   }
-}
 
+  interface TypographyVariants {
+    title: React.CSSProperties;
+    menuTitle: React.CSSProperties;
+  }
+
+  // allow configuration using `createTheme`
+  interface TypographyVariantsOptions {
+    title?: React.CSSProperties;
+    menuTitle: React.CSSProperties;
+  }
+}
+const components = {
```

Comment:
find type here

Reply:
Were able to set it to a subtype of the VariantOptions